.github/workflows/build-lint-test.yml

```diff
@@ -0,0 +1,32 @@
+name: PHP Build, Lint
+
+on:
+    push:
+
```

Comment:
Hey @vigneshk-tw 

Could you please change the trigger condition to PR.

pull_request:
branches:
- master